packages/suite-native-atoms/src/Button/TileButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { Icon, IconName } from '@trezor/icons';
+
+type ActionIconButtonProps = {
+    iconName: IconName;
+    title: string;
+    onPress: () => void;
+    style?: NativeStyleObject;
+};
+
+const rectangleButtonStyle = prepareNativeStyle(utils => ({
+    height: 79,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: utils.borders.radii.small,
+    backgroundColor: utils.colors.gray300,
+    flex: 1,
+}));
+
+export const TileButton = ({ iconName, title, onPress, style }: ActionIconButtonProps) => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <TouchableOpacity style={[applyStyle(rectangleButtonStyle), style]} onPress={onPress}>
+            <Box justifyContent="center" alignItems="center">
+                <Icon name={iconName} />
```

Comment:
What do you think the space should be? I have same amount between them as in the design.

Reply:
Perhaps that's also because they are taller in Figma? (2nd pic)

![image](https://user-images.githubusercontent.com/45338719/174793209-b76e0517-77ba-477f-a151-5f1b86844279.png) 
![image](https://user-images.githubusercontent.com/45338719/174793251-efb9d1b5-939a-4491-83dd-b0aed4759beb.png)

Reply:
https://tppr.me/vF2mV

Reply:
nit: I would vote for margin in those cases in th efuture. Even though they are identical in some cases, semantically padding is for measures within a component and not for spacings between